src/pages/TransactionInfoPage.tsx

```diff
@@ -239,9 +240,33 @@ const TransactionInfoPage = () => {
                   fullPrecision
                 />
               </TableRow>
+              {tokensDeltaAmountsEntries.length > 0 && (
+                <TableRow>
+                  <b>Total Token Amounts</b>
+                  <AlphValuesContainer>
+                    {tokensDeltaAmountsEntries.map(([k, v]) => (
+                      <AlphValue key={k}>
+                        <AddressLink address={k} />
```

Comment:
very nit: How about giving the key/value vars a more descriptive name, like `[addressHash, tokenAmounts]`?